app/posts/apply-for-teacher-training/2020-01-16-question-protocol-for-pilot.md

```diff
@@ -0,0 +1,1722 @@
+    }, {
+      who: ["hesa"],
+      what: "requires this to “create the student's DfE record.”"
+    }],
+    what: ["Retrievable via API"],
+    accuracy: ["Providers will ask to see hard copies for any qualifications awarded"],
+    notes: [
+      "This may be a want, more than a need.",
+      "Not currently shown to providers"
+    ]
+  }
+}) }}
+
+{{ appProtocol({
+  title: "✘ Qualification - (non-UK) country",
+  items: {
+    question: "Country (if outside UK)",
+    required: "Yes (if degree not awarded in UK)",
+    api: {
+      param: "qualifications.degrees[].awarding_body",
+      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#qualification-object"
+    },
+    why: [{
+      who: ["provider"],
+      what: "uses this to perform a NARIC check (they need confirmation from a the institution that they’ve actually obtained the degree they say they have)"
+    }, {
+      who: ["hesa"],
+      what: "requires this to “create the student's DfE record.”"
+    }],
+    what: ["Appears in provider-facing interface and retrievable via API"],
+    accuracy: ["Providers will ask to see hard copies for any qualifications awarded"]
+  }
+}) }}
+
+{{ appProtocol({
+  title: "✘ Qualification - (non-UK) equivalency",
+  items: {
+    question: "Do you know what UK degree your degree is equivalent to?",
+    required: "Yes (if degree not awarded in UK)",
+    api: {
+      param: "qualifications.degrees[].equivalency_details",
+      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#qualification-object"
+    },
+    why: [{
+      who: ["provider"],
+      what: "needs to look for evidence that a qualification is of a standard equivalent to a UK degree"
+    }, {
+      who: ["candidate"],
+      what: "needs to provide evidence that their international qualification demonstrates a similar level and breadth as the equivalent UK qualification"
+    }, {
+      who: ["candidate"],
+      what: "may already have a statement of comparability from NARIC"
+    }],
+    what: ["Appears in provider-facing interface and retrievable via API"],
+    accuracy: ["Providers will ask to see hard copies for any qualifications awarded"],
+    notes: ["Candidates can request a statement of comparability from NARIC, or speak to a TTA who can certify equivalence"]
+  }
+}) }}
+
+## Maths GCSE or equivalent
+
+Candidates can update their answers before (and 5 working days after) submitting their application.
+
+{{ appProtocol({
+  title: "Qualification - type",
+  items: {
+    question: "Type of qualification",
+    required: "Yes",
+    api: {
+      param: "qualifications.gcses[].qualification_type",
+      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#qualification-object"
+    },
+    why: [{
+      who: ["provider"],
+      what: "needs to ensure candidates have achieved a standard equivalent to a grade 4 in GCSE Maths"
```

Comment:
Suggest linking to https://www.gov.uk/government/publications/initial-teacher-training-criteria/initial-teacher-training-itt-criteria-and-supporting-advice#c11-gcse-standard-equivalent